mne/decoding/ssd.py

```diff
@@ -1,15 +1,21 @@
 # Author: Denis A. Engemann <denis.engemann@gmail.com>
 #         Victoria Peterson <victoriapeterson09@gmail.com>
+#         Thomas S. Binns <t.s.binns@outlook.com>
 # License: BSD-3-Clause
 
 import numpy as np
+import scipy as sp
```

Comment:
```suggestion
from scipy import linalg
```

and then use linalg.XXX below

Reply:
Apologies, fixed now!

Reply:
Hi @agramfort, it seems `from scipy import linalg` causes `test_import_nesting.py` to fail. How should I handle this?

Reply:
nest the import inside the function that needs `linalg` like

```
def whatever(...):
    from scipy import linalg
   ...
    u, s, v = linalg.svd(...)
```